.travis.yml

```diff
@@ -58,6 +64,7 @@ before_install:
 install:
   - travis_retry pip install pip setuptools --upgrade
   - travis_retry pip install tox
+  - travis_retry pip install eth_utils tqdm requests
```

Comment:
Since these are only necessary during parity install, we could skip it in all the tests by inserting the `pip install` as a line with its own: `if [ -n "$PARITY_VERSION" ];` just before the `install_parity` command.